packages/connect/src/types.ts

```diff
@@ -11,17 +11,10 @@ export interface Client {
 }
 
 export interface OrderbookChannel {
-    subscribe: (subscriptionOpts: OrderbookChannelSubscriptionOpts, handler: OrderbookChannelHandler) => void;
+    subscribe: (subscriptionOpts: OrderbookChannelSubscriptionOpts) => void;
     close: () => void;
```

Comment:
should the `uri` be discoverable here with a getter? It seems like it would be hard to track down which uri was spitting out errors if you were subscribed to multiple channels.

Reply:
hmm i feel that it does break encapsulation somewhat, perhaps we can have it in the error message itself